### ISSUE TITLE
Two stage fallback silently gets terminated

### ISSUE THREAD

**Issue by kaustuk** [RasaHQ/rasa#7510]
**Description of Problem**:
So basically if we get 2 consecutive times user intent classification less then threshold it will terminated two stage fallback with uttering or trigger any action/utterances. Here is the following example:
User: gibberish(classified less then threshold)
Bot: calls action_default_ask_affirmation 
User:  gibberish(classified less then threshold)
Bot: < does nothing silently terminate loop without raising or calling any action >

**Overview of the Solution**:

So basically here when this case happens we are calling  `_default_deactivation_events` function in loops.py at time to terminate it so we can overwrite this function in TwoStageFallbackAction class.

**Examples** (if relevant):

So in rasa 1.10.x this case was being handled by calling `fallback_nlu_action_name` functions where as this case is not handle in rasa 2.0. Hence it's better to terminate it via some utterances or actions

**Blockers** (if relevant):

**Definition of Done**:

**Comment by kaustuk** [comment]
Also if we agree to have this feature I would like to work on this.

**Comment by kaustuk** [comment]
Sorry my bad this is not the case I haven't gone through the code properly. This is already handled. My bad :)
